### DISCUSSION TITLE
Weird reporting from Sunricher dimmer

### DISCUSSION THREAD

**Discussion by vlinder** [Koenkk/zigbee2mqtt#12211] [Devices]
I have about 6 of this dimmer https://www.zigbee2mqtt.io/devices/ZG2835RAC.html#sunricher-zg2835rac and one of them behaves weirdly.

I have checked the reporting tab and all the devices of the same type seem to have the same reporting configured. Just this one is spamming the logs every 5 seconds. Any ideas?

**Comment by Eggn1n3** [comment]
I assume you checked if they are running the same firmware?
What happens if you unpair the faulty one, power off/on, and repair?

**Reply by vlinder** [reply]
Yes, they are identical in every way I could find through clicking all of the menus.

**Comment by vlinder** [comment]
Managed to solve the spamming by removing the reporting and adding it back with the exact same settings again.. perhaps it could help someone with the same issue.

The reporting was configured as below, before and after..

```
Endpoint: 1
Cluster: haElectricalMeasurement
Attribute: rmsVoltage
Min rep interval: 10
max rep interval: 3600
Min rep change: 1
```
